lib/model/model.go

```diff
@@ -1559,18 +1559,38 @@ func sendIndexTo(minSequence int64, conn protocol.Connection, folder string, fs
 	return maxSequence, err
 }
 
+func (m *Model) makeUsLastModified(fs []protocol.FileInfo) []protocol.FileInfo {
+	fsMod := make([]protocol.FileInfo, 0, len(fs))
+	for _, file := range fs {
+		file.ModifiedBy = uint64(m.id.Short())
```

Comment:
I tried that at first.  Unfortunately it only worked for object adds and modifies.

When I put the code in the location you want it at, then when I delete an object it would report the name of the last modified client as the one who deleted instead of the actual deleter.  I suppose the deletion code doesn't go through walk.go so I tried finding somewhere else I could add this hook also for deletes but never found a clean spot.

Perhaps you know of somewhere else outside walk.go?